File: programmers/js/level00/120895.js
// 인덱스 바꾸기
function solution(my_string, num1, num2) {
  const my_array = [...my_string];
  
  my_array[num1] = my_string[num2];
  my_array[num2] = my_string[num1];
  
  return my_array.join('');
}

// 다른 풀이
// function solution(my_string, num1, num2) {
//   my_string = my_string.split('');
//   [my_string[num1], my_string[num2]] = [my_string[num2], my_string[num1]];
//   return my_string.join('');
// }

File: programmers/java/level00/120895.java
// 인덱스 바꾸기

class Solution {
    public String solution(String my_string, int num1, int num2) {
        String[] myStringArr = my_string.split("");
        
        String temp = myStringArr[num1];
        myStringArr[num1] = myStringArr[num2];
        myStringArr[num2] = temp;
        
        return String.join("", myStringArr);
    }
}